todo_app/app.py

```diff
@@ -1,11 +1,53 @@
-from flask import Flask
+#!
+from flask import Flask, appcontext_popped, render_template, redirect, request
+from flask_config import Config
+from dotenv import load_dotenv, find_dotenv
+from data.trello_items import *
 
-from todo_app.flask_config import Config
+def create_app():
 
-app = Flask(__name__)
-app.config.from_object(Config())
+   file_path = find_dotenv('.env')
+   load_dotenv(file_path)
 
+   token = os.getenv('TRELLO_TOKEN')
+   key = os.getenv('TRELLO_KEY')
+   board_ID = os.getenv('BOARD_ID')
+   url = "https://api.trello.com/1/"
+   url_member = url + "members/carispang"
+   query_string = {"key" : key, "token" : token}
+   
+   app = Flask(__name__)
+   app = Flask(__name__, template_folder='templates')
+   app.config.from_object(Config())
 
-@app.route('/')
-def index():
-    return 'Hello World!'
+   @app.route('/', methods = ['GET', 'POST'])
+   def createUI():
```

Comment:
You don't have to do this, but you'll probably find it easier to work with in the long run if you split the three things this function is currently doing (rendering index.html, creating a todo, updating a todo) into separate routes i.e.

Instead of

```python
   @app.route('/', methods = ['GET', 'POST'])
   def createUI():
        # Code to render index
        if request.method == 'POST':
            if not request.form.get('complete_button'):
                # Code to create cards
            else:
                # Code to update cards
   return render_template('index.html', ls = todo_cards)
```

do
```python
@app.route('/', methods = ['GET'])
def index():
    # Code to render index
    return render_template('index.html', ls = todo_cards)

@app.route('/createTodo', methods = ['POST'])
    # Code to create cards
    return redirect('/')

@app.route('/markComplete', methods = ['POST'])
    # Code to update cards
    return redirect('/')
```

You'll have to update `index.html` with the new routes as well.